File: examples/availability/retrieve_ipfs/retrieve_ipfs.go
package main

import (
	"log"
	"os"

	bloock "github.com/bloock/bloock-sdk-go/v2"
	"github.com/bloock/bloock-sdk-go/v2/client"
	"github.com/bloock/bloock-sdk-go/v2/entity/availability"
)

func main() {
	bloock.ApiKey = os.Getenv("API_KEY")

	recordClient := client.NewRecordClient()

	id := "publish uuid result"
	_, err := recordClient.FromLoader(availability.NewIpfsLoader(id)).Build()
	if err != nil {
		log.Println(err)
	}
}

File: examples/availability/retrieve_ipfs/RetrieveIpfs.java
import com.bloock.sdk.client.RecordClient;
import com.bloock.sdk.entity.availability.IpfsLoader;
import com.bloock.sdk.entity.record.Record;

public class RetrieveIpfs {
  public static void main(String[] args) throws Exception {
    RecordClient recordClient = new RecordClient();

    String id = "publish uuid result";
    Record record = recordClient.fromLoader(new IpfsLoader(id)).build();
  }
}